stable/centrifugo/templates/deployment.yaml

```diff
@@ -1,14 +1,15 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
```

Comment:
did you test with an existing deployment and then upgrade to this release? is it not going to break?

Reply:
Yes, just tested on k8s 1.15 cluster, upgrade works well.
Note that current version does not work in 1.16+ clusters due to api deprecation
https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/